app/assets/stylesheets/application.scss

```diff
@@ -32,16 +32,49 @@
  // }
 
 .jumbotran {
-    position: relative;
-    background-color: $pastel-grey;
-    width: 100%;
-    height: 100%;
-    background-size: cover;
-    overflow: hidden;
-
-    border-radius: 25px;
-    border: 2px solid $pastel-orange;
-    padding: 20px;
+  position: relative;
```

Comment:
Properties should be ordered background-color, background-size, border, border-radius, height, overflow, padding, position, width